EIPS/eip-6785.md

```diff
@@ -0,0 +1,203 @@
+---
+eip: 6785
+title: ERC-721 Utilities Extension
+description: Easy access to utility information via functions and the NFT's metadata
```

Comment:
The description as written doesn't give much more information than the title. If you can cram what a "utility function" is in here I think it would be very helpful.